NEXT_CHANGELOG.md

```diff
@@ -3,6 +3,10 @@
 ## Release v0.259.0
 
 ### Notable Changes
+* Diagnostics messages are no longer buffered to be printed at the end of command, flushed after every mutator ([#3175](https://github.com/databricks/cli/pull/3175))
+* Diagnostics are not always rendered with forward slashes in file paths, even on Windows ([#3175](https://github.com/databricks/cli/pull/3175))
+* "bundle summary" now prints diagnostics to stderr not to stdout in non-json mode ([#3175](https://github.com/databricks/cli/pull/3175))
```

Comment:
```suggestion
* "bundle summary" now prints diagnostics to stderr instead of stdout in text output mode ([#3175](https://github.com/databricks/cli/pull/3175))
```